.scalafix3.conf

```diff
@@ -4,6 +4,7 @@ rules = [
 
 OrganizeImports.groupedImports = Explode
 OrganizeImports.expandRelative = true
+OrganizeImports.removeUnused = false
```

Comment:
This is because warn unused doesn't work with Scala 3 right?

Reply:
yep + by default it's `true` that fails the internal validation for scala3 projects